spec/integration/parser_process_spec.rb

```diff
@@ -31,6 +31,7 @@
     it 'returns nil for unused schemas' do
       schemas = invalid_schemas
       schemas.each do |schema|
+        Item::Content::Parser.extract_content(schema)
```

Comment:
Is this needed? It's called in the following line

Reply:
Good spot I forget to remove that line